### ISSUE TITLE
Feautre Request: Edit/Duplicate

### ISSUE THREAD

**Issue by t-oster** [Elinvention/efiboots#35]
Hi,

nice UI. I would vote for this to be integrated in the system settings of ubuntu etc (I use arch btw ;))

However, since I have a long list of parameters I am afraid to break, it would be great if I could edit existing entries with the UI. If that is not so easy to implement, at least a duplicate button, which just prefills the "new entry" UI with the values of the currently selected entry would be nice.

**Comment by Elinvention** [comment]
That's a good idea. In the longer run, I was thinking to rewrite the GUI using libadwaita, remove GtkTreeView (it' buggy and ugly) and switch to some cooler widget with sleek animations. That should make all sorts of operations on entries easier and better looking. Things like drag and drop to reorder entries, one click next boot selection, showing and hiding details of entries, editing and so on.

**Comment by t-oster** [comment]
Sounds good. However nice animations etc are not so important to me as an edit option. I mean most of the time I am fiddling with the kernel boot parameters and if I broke something, I want to edit it and try again.

**Comment by Elinvention** [comment]
Yes, I mean, nice animations should come for free with libadwaita. The important thing is to improve usability.
